src/module/migration/migrations/937-remove-invalid-aura-traits.ts

```diff
@@ -0,0 +1,42 @@
+import { ItemSourcePF2e } from "@item/base/data/index.ts";
+import { actionTraits, effectTraits } from "@scripts/config/traits.ts";
+import { MigrationBase } from "../base.ts";
+
+export class Migration937RemoveInvalidAuraTraits extends MigrationBase {
+    static override version = 0.937;
+
+    static #_invalidTraits: Set<string> | null = null;
+
+    /**
+     * Computes and returns a list of all invalid traits.
+     * Doing it this way avoids us accidentally removing homebrew effect traits.
+     */
+    static get #invalidTraits(): Set<string> {
+        if (this.#_invalidTraits) return this.#_invalidTraits;
+
```

Comment:
```suggestion
```
leave some whitespace for the sick children to eat

Reply:
How wasteful of me